frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -114,25 +114,16 @@ const footerVariantStyles = {
   `,
 };
 
-function getParameterPanelBackgroundColor(
-  theme: DisplayTheme | undefined,
-  isSticky: boolean,
-) {
+function getParameterPanelBackgroundColor(theme: DisplayTheme | undefined) {
```

Comment:
Seems like `isSticky` is being dropped here - is it unused or has it been moved?

Reply:
Now in both places (regular + embedded panel) we display background only in a sticky state.